test/support/WithWireMockHBase.scala

```diff
@@ -5,19 +5,37 @@ import com.github.tomakehurst.wiremock.client.{ MappingBuilder, ResponseDefiniti
 import org.scalatest.Suite
 import play.api.http.Status.OK
 import play.mvc.Http.MimeTypes.JSON
+
 import repository.hbase.HBase.rowKeyUrl
 import repository.hbase.LocalUnitRowKey
 import uk.gov.ons.sbr.models.Period
 import uk.gov.ons.sbr.models.enterprise.Ern
 import uk.gov.ons.sbr.models.localunit.Lurn
 
+import repository.hbase.unit.enterprise.EnterpriseUnitRowKey
+
 trait WithWireMockHBase extends WithWireMock with BasicAuthentication with HBaseResponseFixture { this: Suite =>
   override val wireMockPort = 8075
 
+  private val ColumnFamily = "d"
+  private val Namespace = "sbr_control_db"
+
+  private def createUrlAndThenGetHbaseJson(tableName: String, rowKey: String) =
+    getHbaseJson(
```

Comment:
While it doesn't comply with usual camel-case conventions, I have been trying to use "HBase" as that is how the project itself capitalises the name.